addons/web_editor/static/src/js/wysiwyg/widgets/image_crop_widget.js

```diff
@@ -92,8 +92,9 @@ const ImageCropWidget = Widget.extend({
         this._onDocumentKeydown = this._onDocumentKeydown.bind(this);
         // We use capture so that the handler is called before other editor handlers
         // like save, such that we can restore the src before a save.
-        this.document.addEventListener('mousedown', this._onDocumentMousedown, {capture: true});
-        this.document.addEventListener('keydown', this._onDocumentKeydown, {capture: true});
+        this.cropperDocument = this.$el[0].ownerDocument;
```

Comment:
intead of using `this.cropperDocument` it better we keep it simple and just use `this.ownerDocument`.

Reply:
@shsa-odoo Currently it is the same as you mentioned, this.document is in fact ownerDocument. But in website ownerDocument of image is different than ownerDocument of cropper widget. That is the reason why event is not getting caught in website. In order to trigger both of the events we need cropper's owner document. In notes we have only one document hence we have no problem there.